easybuild/easyconfigs/b/BCFtools/BCFtools-1.3-foss-2016a.eb

```diff
@@ -0,0 +1,40 @@
+easyblock = 'ConfigureMake'
+
+name = 'BCFtools'
+version = '1.3'
+
+homepage = 'http://www.htslib.org/'
+description = """Samtools is a suite of programs for interacting with high-throughput sequencing data.
+ BCFtools - Reading/writing BCF2/VCF/gVCF files and calling/filtering/summarising SNP and short indel sequence variants"""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+toolchainopts = {'optarch': True, 'pic': True}
+
+sources = [SOURCELOWER_TAR_BZ2]
+source_urls = ['https://github.com/samtools/%(namelower)s/releases/download/%(version)s']
+
+patches = ['%(name)s-%(version)s_extHTSlib_Makefile.patch']
+
+dependencies = [
+    ('HTSlib', '1.3'),
+    ('zlib', '1.2.8'),
+    ('GSL', '1.16'),
```

Comment:
@RvDijk we have existing easyconfigs for BCFtools that use GSL 2.1, any reason you're sticking to 1.16 here?